src/test/java/org/example/controller/AuthControllerTest.java

```diff
@@ -0,0 +1,55 @@
+package org.example.controller;
+
+import org.example.controllers.AuthController;
+import org.example.exception.AuthDaoException;
+import org.example.exception.InvalidCredentialsException;
+import org.example.models.LoginRequest;
+import org.example.services.AuthService;
+import org.junit.jupiter.api.Test;
+
+import javax.ws.rs.core.Response;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class AuthControllerTest {
+    AuthService authService = mock(AuthService.class);
+    AuthController authController = new AuthController(authService);
+
+    String jwtToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9."
+                    + "eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkJhZWxkdW5nIFVzZXIiLCJpYXQiOjE1MTYyMzkwMjJ9."
+                    + "qH7Zj_m3kY69kxhaQXTa-ivIpytKXXjZc1ZSmapZnGE";
+
+    LoginRequest loginRequest = new LoginRequest("user1", "user1");
+
+    @Test
+    public void login_shouldReturnJwtTokenInResponse_whenServiceReturnsJwtToken()
+            throws InvalidCredentialsException, AuthDaoException {
+        when(authService.login(loginRequest)).thenReturn(jwtToken);
+
+        Response response = authController.login(loginRequest);
+        assertEquals(200, response.getStatus());
+        assertEquals(jwtToken, response.getEntity());
+    }
+
+    @Test
+    public void login_shouldReturn400Status_whenServiceThrowsInvalidCredentialsException()
```

Comment:
This test will need to change to 401 status

Reply:
Resolved